### PR TITLE
Use group/uuid/roles API in group detail

### PR DESCRIPTION
**Pull request by PanSpagetka**

### CODE REVIEW

**Comment by PanSpagetka** [comment]
ping @karelhala

**Comment by karelhala** [comment]
I see some kind of strange pagination in group detail
Screenshot from 2019-12-09 16-24-50

Might want to fetch roles for curr group as well to get full pagination

**Comment by karelhala** [comment]
@PanSpagetka I don't know if it is part of this PR or not, but in past you were able to see selected roles in group detail -> add new role dialog. Now it looks like it's gone

This group
Screenshot from 2019-12-09 16-35-34

Should have this role selected
Screenshot from 2019-12-09 16-35-26

**Comment by PanSpagetka** [comment]
I found https://marvelapp.com/20073f2g/screen/63956552 it seems that already selected roles shouldn't be present in add group dialog. I will change it so it is aligned with mocks.

File: src/smart-components/group/role/group-roles.js [hunk 1]
```diff
@@ -61,15 +60,12 @@ const GroupRoles = ({
...
-      ...userIdentity && userIdentity.user && userIdentity.user.is_org_admin ?
-        [
-          {
-            title: 'Remove from group',
-            onClick: (_event, _rowId, role) => {
-              removeRoles(uuid, [ role.uuid ]);
-            }
-          }
-        ] : []
+      {
```
Comment (karelhala) [inline src/smart-components/group/role/group-roles.js]: Please include `...userIdentity && userIdentity.user && userIdentity.user.is_org_admin ?` in here
```JS
        ...userIdentity && userIdentity.user && userIdentity.user.is_org_admin ?
        [
          title: 'Remove from group',
          onClick: (_event, _rowId, role) => {
            removeRoles(uuid, [ role.uuid ], () => fetchRolesForGroup(pagination)(uuid));
          }
        ]
```

File: src/smart-components/group/role/group-roles.js [hunk 2]
```diff
@@ -118,12 +115,11 @@ const GroupRoles = ({
         createRows={ createRows }
         data={ roles }
         filterValue={ filterValue }
-        fetchData={ (config) => fetchRoles(mappedProps(config)) }
+        fetchData={ (config => fetchRolesForGroup(config)(uuid)) }
```
Comment (karelhala) [inline src/smart-components/group/role/group-roles.js]: No need to wrap in `()`

**Pull request merged**